src/routes.ts

```diff
@@ -0,0 +1,43 @@
+import express from "express";
+import {param, query} from 'express-validator'
+import {getChainNames} from './utils/chainUtils'
+import {returnIfCached} from './middleware/returnIfCached'
+import {validationCheck} from './middleware/validationCheck'
+import {totalVolumeController} from './controllers/totalVolumeController'
+import {totalTxnCountController} from './controllers/totalTxnCountController'
+import {newUsersController} from './controllers/newUsersController'
+
+let defaultMiddleware = [
+    validationCheck,
+    returnIfCached
+]
+
+const router = express.Router();
+
+router.get('/volume/total/:direction',
+    [
+        param('direction').isIn(["in", "out"])
+    ],
+    ...defaultMiddleware,
```

Comment:
Is it possible to pass these default middlewares to the express router on line 15?

Reply:
I'd like to, but order of middlewares is critical, express-validator needs to go before any middleware. I could probably use the caching middleware in the router, but that doesn't give us flexiblity on other endpoints...